### PR TITLE
improvement/loading-mini

### PR DESCRIPTION
**Pull request by seesleestak**
Addressing issue #146

### CODE REVIEW

File: packages/input/src/Input.js [hunk 1]
```diff
@@ -22,23 +22,18 @@ export const SuffixContainer = styled.span`
 `
 
 export const SpinnerContainer = SuffixContainer.extend`
-  top: 12px;
+  top: ${({ mini }) => (mini ? '5px' : '12px')};
```
Comment (billyxs) [inline packages/input/src/Input.js]: You like 5px instead of 6px?

Comment (seesleestak) [reply packages/input/src/Input.js]: It was a toss up between them, if I’m honest. If you like 6px, that’s good too :)

Comment (billyxs) [reply packages/input/src/Input.js]: Wasn't sure with your new linux browser experience 😉 It is a bit on the odd side either way.

Comment (seesleestak) [reply packages/input/src/Input.js]: :laughing:

**Pull request merged**